src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -702,9 +702,13 @@ byte getServerSupportedDataClassificationVersion() {
     // Boolean that indicates whether LOB objects created by this connection should be loaded into memory
     private boolean delayLoadingLobs = SQLServerDriverBooleanProperty.DELAY_LOADING_LOBS.getDefaultValue();
 
-    boolean getDelayLoadingLobs() {
+    public boolean getDelayLoadingLobs() {
```

Comment:
public methods should have description header

Reply:
Shouldn't you override here?

Reply:
Well there's no previous implementation to get confused about, so the only reason to do it would be for "commenting".